src/com/lftechnology/java_training/jan5/ContentInArraysComparator.java

```diff
@@ -0,0 +1,106 @@
+package com.lftechnology.java_training.jan5;
+
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * Compare if two arrays have the same content (doesn't matter if they aren't in order)
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class ContentInArraysComparator {
+
+	public static void main(String[] args) {
+		ArrayStore arrayStore1 = new ArrayStore();
+		arrayStore1.setValueForArray("Enter value for array1");
+
+		ArrayStore arrayStore2 = new ArrayStore();
+		arrayStore2.setValueForArray("Enter value for array2");
+
+		arrayStore1.compareArrayValues(arrayStore2);
+	}
+
+}
+
+/**
+ * this class stores value in array and compares two array if they both have same content or not despite of random order
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+class ArrayStore {
+	private String[] arrayOfValues = new String[10];
+	private final static Logger LOGGER = Logger.getLogger(ArrayStore.class.getName());
+	private Scanner input = new Scanner(System.in);
+	private int countValuesInArray = 0;
+	private final String Y = "y";
+	private final String YES = "yes";
```

Comment:
when a attribute is associated with Class and is final then it is better to declare it as static final. In your case it'll be private static final String Y = "y"; The same reason as Logger is declared.